cx_Freeze/macdist.py

```diff
@@ -47,7 +47,7 @@ def buildDMG(self):
                 raise OSError('creation of Applications shortcut failed')
 
             createargs.append('-srcfolder')
-            createargs.append(self.buildDir + '/Applications')
+            createargs.append(os.path.join(self.buildDir, '/Applications'))
```

Comment:
typo
os.path.join(self.buildDir, 'Applications')

Reply:
This is weird my git diff and editor report this
```
diff --git a/cx_Freeze/macdist.py b/cx_Freeze/macdist.py
index aa48f0c..dae37b5 100644
--- a/cx_Freeze/macdist.py
+++ b/cx_Freeze/macdist.py
@@ -47,7 +47,7 @@ class bdist_dmg(Command):
                 raise OSError('creation of Applications shortcut failed')

             createargs.append('-srcfolder')
-            createargs.append(self.buildDir + '/Applications')
+            createargs.append(os.path.join(self.buildDir, '/Applications'))
```

Reply:
yes, but should without the '/' no?

Reply:
Oh I misunderstood thanks

Reply:
Ok that was embarrassing but I think I got them all :)